README.md

```diff
@@ -49,6 +49,7 @@ make test
 - Provided HTTP healtcheck will be transferred to Consul.
 - Labels with `tag` value will be converted to Consul tags, `marathon` tag is added by default
  (e.g, `labels: ["public":"tag", "varnish":"tag", "env": "test"]` → `tags: ["public", "varnish", "marathon"]`).
+- To schedule Marathon-consul sync one of two options has to be set: `sync-leader` (run sync only on specified node [leadership](https://mesosphere.github.io/marathon/docs/rest-api.html#get-v2-leader)) or `sync-force` (run always).
```

Comment:
IMO we should describe what's the difference between `sync-leader` and `marathon-location`. Also describe when sync will take place so I'll write something like this:
> (run sync only on node that is Marathon leader ([marthon leader] equals `sync-leader`)